package/MDAnalysis/analysis/nuclinfo.py

```diff
@@ -131,8 +132,10 @@ def wc_pair(universe, i, bp, seg1="SYSTEM", seg2="SYSTEM"):
         a1, a2 = "N3", "N1"
     if universe.select_atoms(" resid {0!s} ".format(i)).resnames[0] in ["DG", "DA", "A", "G", "ADE", "GUA"]:
         a1, a2 = "N1", "N3"
-    wc_dist = universe.select_atoms(
-        " (segid {0!s} and resid {1!s} and name {2!s})  or (segid {3!s} and resid {4!s} and name {5!s}) ".format(seg1, i, a1, seg2, bp, a2))
+    wc_dist = universe.select_atoms("(segid {0!s} and resid {1!s} "
```

Comment:
I could do that too. With the latest commit, I cut after the `or` so it make sense with the logic of the string. I still did not cut after the parenthesis so the lines are slightly too long, but 87 columns should still be good enough.